### PR TITLE
test: help refactor tests

### PR DESCRIPTION
**Pull request by JoanFM**
**Changes introduced**
Fix 2 more tests.

### CODE REVIEW

File: jina/drivers/search.py [hunk 1]
```diff
@@ -127,9 +127,8 @@ def _apply_all(self, docs: 'DocumentSet', *args, **kwargs) -> None:
         for doc, topks, scores in zip(doc_pts, idx, dist):
 
             topk_embed = fill_fn(topks) if (self._fill_embedding and fill_fn) else [None] * len(topks)
-
             for match_hash, score, vec in zip(topks, scores, topk_embed):
-                r = doc.add_match(doc_id=match_hash,
+                r = doc.add_match(doc_id=uid.hash2id(match_hash),
```
Comment (hanxiao) [inline jina/drivers/search.py:131]: i remember I add shortcut for hash-value `doc_id` in `add_match`, let me check

Comment (hanxiao) [reply jina/drivers/search.py:131]: image
yes,  I think this is not needed

Comment (hanxiao) [inline jina/drivers/search.py:131]:
```suggestion
                r = doc.add_match(doc_id=match_hash,
```

**Review by nan-wang — COMMENTED** [review]
We need to rename the helper function after the `random_docs` is deprecated. `random_docs_new_api` does not sounds a good name.  🤓

**Pull request merged**